src/app/Root.js

```diff
@@ -35,8 +44,41 @@ const Root = () => (
           <ConnectedRouter history={history}>
             <RouterScrollToTop>
               <Switch>
-                <Route path="/auth" component={AuthApp} />
-                <Route exact path="/" component={IsLoggedIn(AccountApp)} />
+                <Route
+                  path="/auth"
+                  component={() => (
+                    <AuthScreen>
```

Comment:
why moving all this into the Root? Dont we want to have `features/auth/routes` or something? In a real app this would become really huge. Starter should show how to do stuff in a real app, so if we think this should be split, we should split it even though it's ok to have here for the starter.

Reply:
starter should be leading by example, setting the "best practice"

Reply:
i am not sure how to proceed here. I more or less agree with your idea, but still moving it all to Root does not seem right to me. At least we should have root.js and routes.js for this

Reply:
Yeah, let's hide/group it back under /auth/routes. This is inconsistent with rootSaga, which should be somewhat similar to Root.js I guess.

If we wanted all the routes in the Root, then we should break the auth ducks into login ducks, signup ducks etc... too much work for now